src/metadata/StreamMetadata.ts

```diff
@@ -4,11 +4,29 @@
  * See file LICENSE or go to https://spdx.org/licenses/AGPL-3.0-or-later.html for full license details.
  */
 
-import { WebSocketReliable, Connect, EventEmitter, Util } from '@ceeblue/web-utils';
+import { WebSocketReliable, Connect, EventEmitter, Util, WebSocketReliableError } from '@ceeblue/web-utils';
 import { MTrack, MType, Metadata } from './Metadata';
 
 const sortByMAXBPS = (track1: MTrack, track2: MTrack) => track2.maxbps - track1.maxbps;
 
+export enum StreamState {
+    UNKNOWN = '',
+    ONLINE = 'Stream is online',
+    OFFLINE = 'Stream is offline',
+    LOADING = 'Stream is initializing',
```

Comment:
why **LOADING** not **initializing**

Reply:
https://github.com/CeeblueTV/webrtc-client/pull/19/commits/c3bc7ba05a1871b05e646e82e0327b0fc0b10894